### ISSUE TITLE
Some tests fail on Windows due to different DIRECTORY_SEPERATOR

### ISSUE THREAD

**Issue by jankal** [ApiGen/ApiGen#804]
```
C:\xampp\php\php.exe -dxdebug.coverage_enable=1 "C:/Users/Alexander Jank/Projekte/ApiGen/vendor/phpunit/phpunit/phpunit" --coverage-clover "C:\Users\Alexander Jank\.PhpStorm2016.3\system\coverage\ApiGen$Unnamed.coverage" --configuration "C:\Users\Alexander Jank\Projekte\ApiGen\phpunit.xml.dist" --teamcity
Testing started at 22:35 ...
PHPUnit 6.0.8 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.1.1 with Xdebug 2.5.1
Configuration: C:\Users\Alexander Jank\Projekte\ApiGen\phpunit.xml.dist


Method Mockery_5_Symfony_Component_Console_Output_OutputInterface::writeln() does not exist on this mock object
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Console\Command\GenerateCommand.php:219
 C:\Users\Alexander Jank\Projekte\ApiGen\tests\MethodInvoker.php:21
 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Console\Command\GenerateCommandExecuteTest.php:54
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Console\Command\GenerateCommandPrepareOptionsTest.php:55
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\src
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/src
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Console\Command\GenerateCommandPrepareOptionsTest.php:70
 

Failed asserting that two strings are identical.
Expected :entities/Category.php
Actual   :ntities/Category.php
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Generator\Resolvers\RelativePathResolverTest.php:66
 

Could not determine "C:\Users\Alexander Jank\Projekte\ApiGen\tests\Generator\TemplateGenerators\SourceCodeSource\SomeClass.php" relative path
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Generator\Resolvers\RelativePathResolver.php:43
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Generator\TemplateGenerators\SourceCodeGenerator.php:127
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Generator\TemplateGenerators\SourceCodeGenerator.php:84
 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Generator\TemplateGenerators\SourceCodeGeneratorTest.php:72
 

Could not determine "C:\Users\Alexander Jank\Projekte\ApiGen\tests\Generator\TemplateGenerators\SourceCodeSource\SomeClass.php" relative path
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Generator\Resolvers\RelativePathResolver.php:43
 C:\Users\Alexander Jank\Projekte\ApiGen\src\Generator\TemplateGenerators\SourceCodeGenerator.php:127
 C:\Users\Alexander Jank\Projekte\ApiGen\tests\MethodInvoker.php:21
 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Generator\TemplateGenerators\SourceCodeGeneratorTest.php:85
 

This test did not perform any assertions

This test did not perform any assertions

This test did not perform any assertions

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/index.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/index.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:73
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/namespace-MyNamespace.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/namespace-MyNamespace.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:91
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/package-MyPackage.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/package-MyPackage.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:100
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/class-SomeClass.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/class-SomeClass.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:112
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/constant-SomeConstant.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/constant-SomeConstant.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:124
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/function-SomeFunction.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/function-SomeFunction.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:136
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests/temp/38884/api/source-code-SomeClass.html
Actual   :C:/Users/Alexander Jank/Projekte/ApiGen/tests/temp/38884/Api/source-code-SomeClass.html
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Templating\TemplateNavigatorTest.php:148
 

Failed asserting that two strings are identical.
Expected :C:\Users\Alexander Jank\Projekte\ApiGen\tests\Theme/ThemeConfigPathResolverSource/config.neon
Actual   :C:\Users\Alexander Jank\Projekte\ApiGen\tests\Theme/ThemeConfigPathResolverSource//config.neon
 <Click to see difference>

 C:\Users\Alexander Jank\Projekte\ApiGen\tests\Theme\ThemeConfigPathResolverTest.php:17
 
Der Befehl "chmod" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.

Failed asserting that exception of type "\ApiGen\Utils\Neon\Exceptions\FileNotReadableException" is thrown.


Time: 1.21 minutes, Memory: 68.00MB


ERRORS!
Tests: 531, Assertions: 794, Errors: 3, Failures: 12, Risky: 3.

Generating code coverage report in Clover XML format ... done

Process finished with exit code 2
```

**Comment by TomasVotruba** [comment]
Hi, thanks for report.

Could you fix it on your system and send PR please?

**Comment by ek9** [comment]
From what I see in the failures, it seems the tests complain due to using non-windows like directories. Could you try running your tests via this command instead, just to see if any of those failure are due to providing a non-standard path (you are using `/` instead of `\` as a separator):

C:\xampp\php\php.exe -dxdebug.coverage_enable=1 "C:\Users\Alexander Jank\Projekte\ApiGen\vendor\phpunit\phpunit\phpunit" --coverage-clover "C:\Users\Alexander Jank\.PhpStorm2016.3\system\coverage\ApiGen$Unnamed.coverage" --configuration "C:\Users\Alexander Jank\Projekte\ApiGen\phpunit.xml.dist" --teamcity

**Comment by ek9** [comment]
This is now fixed on `4.2.x-dev`.

**Comment by jankal** [comment]
@ek9 Thanks for handling this. Have been remote with only my mobile the last week.
